assets/containerisableComponents/payPalContributionButton/payPalContributionButton.jsx

```diff
@@ -25,11 +28,12 @@ type PropTypes = {
   isoCountry: IsoCountry,
   countryGroupId: CountryGroupId,
   errorHandler: (string) => void,
-  canClick?: boolean,
-  buttonText?: string,
-  additionalClass?: string,
-  inPaymentLogosTest?: boolean,
-  onClick?: ?(void => void),
+  canClick: boolean,
```

Comment:
Were these all turned into required values intentionally? I know we went around that loop a few times yesterday but I thought you'd made it work with optional values in the end. Just wanted to double-check.

Reply:
This is the flow vs ESLint shootout: "Tidied up some flow/ESlint issues with default props. Basically, they fight each other. ESLint wants you to put the optional ? marker on the prop and flow doesn't. I've gone with flow in this case because it gives us type safety."